routes/personality_route.py

```
@@ -1,275 +1,160 @@
-        "personality_tags": tags_list,
-        "created_at": str(row["created_at"])
-    }
-    ```
-    """
+# ────────────────────────────────────────────────
+# 최신 성향 조회
+# ────────────────────────────────────────────────
+@personality_router.get("/analysis", response_model=MBTI)
+def get_user_mbti(
+    user_id: str = Depends(verify_token),
+    db: Session = Depends(get_db),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)